src/audio/google/google_rtc_audio_processing.c

```diff
@@ -526,7 +535,7 @@ static int google_rtc_audio_processing_init(struct processing_module *mod)
 	cd->num_frames = NUM_FRAMES;
 
 	/* Giant blob of scratch memory. */
-	GoogleRtcAudioProcessingAttachMemoryBuffer(sys_cache_cached_ptr_get(&aec_mem_blob[0]),
+	GoogleRtcAudioProcessingAttachMemoryBuffer(cached_ptr(&aec_mem_blob[0]),
```

Comment:
could we not just put a linter disable here for sparse?

Reply:
> could we not just put a linter disable here for sparse?

I thought a "linter disable" was what this PR is already doing :-D

More seriously, what do you mean by "here"? This file, this directory, other?

How about something like this below? Now you could really rid all this code of all sparse annotations.

At this point it would better to create a new `app/overlays/sparse.conf`, similar to #9264 which I just submitted.

```diff
--- a/.github/workflows/sparse-zephyr.yml
+++ b/.github/workflows/sparse-zephyr.yml
@@ -70,6 +70,7 @@ jobs:
              ./sof/zephyr/docker-build.sh ${{ matrix.platform }}          \
              --cmake-args=-DZEPHYR_SCA_VARIANT=sparse --cmake-args=-DCONFIG_LOG_USE_VLA=n  \
              --cmake-args=-DCONFIG_MINIMAL_LIBC=y  \
+             --cmake-args=-DCONFIG_GOOGLE_RTC_STUFF=n  \
              --pristine 2>&1 | tee _.log
 
              printf '\n\n\t\t\t  ---- Messages below are treated as sparse errors --- \n\n\n'

Reply:
No no, we want the sparse checking in general in this file.  It was this one situation where we're dealing with buffers (in cached addresses) that need to be passed to a third party library without sparse annotation that something needs to be fudged.  I think there's room for argument about where the fudging should be (I lean hard to the "brevity" side over "pedantic correctness" in this patch).  But all the other SOF API use wants to know that it will be cache-correct going forward.

Reply:
Ah got it, wasn't sure if we could do something like a style linter and drop a comment to silence this one line